smoke-tests/features/step-definitions/linkingPages.steps.js

```diff
@@ -0,0 +1,49 @@
+const { Given, When, Then } = require("cucumber");
+const AddLinkSection = require("../pageobjects/sections/add-link.section");
+const AddPageSection = require("../pageobjects/sections/add-page.section");
+const FormDesignerPage = require("../pageobjects/pages/form-designer.page");
+const MenuSection = require("../pageobjects/sections/menu.section");
+const Actions = require("../actions/actions");
+const { acceptAlert } = require("../../support/testHelpers");
+
+Given("I have chosen to {string} to my form", (menuOption) => {
+  Actions.createNewConfig();
+  MenuSection.buttonByName(menuOption).click();
+});
+
+When("I link ths page to link from the {string}", (linkedPage) => {
+  this.newPageName = "Second page";
+  AddPageSection.linkFrom(linkedPage);
+  AddPageSection.pageTitle.setValue(this.newPageName);
+  AddPageSection.saveBtn.click();
+});
+
+Then("my page is created with a link to the page", () => {
+  this.pageName = this.newPageName.toLowerCase().replace(" ", "-");
```

Comment:
This is how we are sanitising for URLs, it's in designer/client/helpers. feel free to lift. If you're comfortable with refactoring it into model/helpers, you can do so or you can do the ol copy + paste and put a `//TODO:` 

```
export function toUrl(title) {
  return `/${(title?.replace(/[^a-zA-Z0-9- ]/g, "") ?? "")
    .trim()
    .replace(/ +/g, "-")
    .toLowerCase()}`;
}
```

Reply:
These lines of code are used to locate the page by CSS locator as the ID's in the HTML are displayed as follows: id="/first-page"